helpers.go

```diff
@@ -200,3 +201,18 @@ type runOptFunc func(*llb.ExecInfo)
 func (f runOptFunc) SetRunOption(ei *llb.ExecInfo) {
 	f(ei)
 }
+
+// ProgressGroup creates a progress group with the given name.
+// If a progress group is already set in the constraints the id is reused.
+// If no progress group is set a new id is generated.
+func ProgressGroup(name string, weak bool) llb.ConstraintsOpt {
```

Comment:
Just for my own understanding can you possibly explain the reasoning for this change? Why is better to reuse the ID of a progress group if one already exists?

Reply:
It lets you use the same progress group but add a new message to it.

Reply:
I think I answered my own question. It looks like we're being more careful now about threading a progress group through to all the rpm building steps, and making sure everything is under the same progress group

Reply:
Whoops didn't see your response